src/app/components/developer/developer.html

```diff
@@ -205,16 +205,28 @@ <h2 class="panel-title">Editing - {{ $ctrl.developer.name }}</h2>
     </span>
 
     <span feature-flag="ui-bump-major">
+      <h1>{{ $ctrl.developer.name }}</h1>
       <div class="panel panel-default">
         <div class="panel-heading"
              ng-if="!$ctrl.isChangeRequest">
-          <h2 class="panel-title">{{ $ctrl.developer.name }}</h2>
+          <h2 class="panel-title">Edit Developer</h2>
```

Comment:
I know they wanted the name above, but if we do it this way the page will have two "h1"s, one with the phrase "Edit Developer", and the second with the developer's name, and then the phrase "edit developer". We shouldn't have two "h1"s, and having the name on the containing page update when the model changes causes challenges with cancellation / save errors. Not that we can't do it, but maybe not worth the effort